### PR TITLE
changed all broken URL of repo

### PR DESCRIPTION
**Pull request by ShahriarShafin**
hope this will help you !

### CODE REVIEW

File: api/docs/swagger.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
         "contact": {},
         "license": {
             "name": "GNU General Public License v3.0",
-            "url": "https://github.com/sundowndev/PhoneInfoga/blob/master/LICENSE"
+            "url": "https://github.com/sundowndev/phoneinfoga/blob/master/LICENSE"
```
Comment (sundowndev) [inline api/docs/swagger.json:13]: This file was automatically generated, please do not edit it

File: api/docs/docs.go [hunk 2]
```diff
@@ -21,7 +21,7 @@ var doc = `{
         "contact": {},
         "license": {
             "name": "GNU General Public License v3.0",
-            "url": "https://github.com/sundowndev/PhoneInfoga/blob/master/LICENSE"
+            "url": "https://github.com/sundowndev/phoneinfoga/blob/master/LICENSE"
```
Comment (sundowndev) [inline api/docs/docs.go:24]: This file was automatically generated, please do not edit it

File: api/docs/swagger.yaml [hunk 3]
```diff
@@ -125,7 +125,7 @@ info:
   description: Advanced information gathering & OSINT framework for phone numbers.
   license:
     name: GNU General Public License v3.0
-    url: https://github.com/sundowndev/PhoneInfoga/blob/master/LICENSE
+    url: https://github.com/sundowndev/phoneinfoga/blob/master/LICENSE
```
Comment (sundowndev) [inline api/docs/swagger.yaml:128]: This file was automatically generated, please do not edit it

**Review by sundowndev — COMMENTED** [review]
Thanks for that, I'll run go generate in your branch to ensure nothing got overridden.

**Review by sundowndev — APPROVED** [review]
Generated file are OK

LGTM

**Comment by ShahriarShafin** [comment]
Thanks.

**Pull request merged**